core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/datalake/dr/backup/AbstractDatabaseBackupAction.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.dr.backup;
+
+import com.sequenceiq.cloudbreak.common.event.Payload;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.dr.BackupRestoreContext;
+import com.sequenceiq.flow.core.AbstractAction;
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.core.FlowState;
+
+public abstract class AbstractDatabaseBackupAction<P extends Payload>
```

Comment:
The code isn't technically redundant, because one has the signature `getFailurePayload(DatabaseBackupSuccess payload...)` and the other has the signature `getFailurePayload(DatabaseBackupFailedEvent payload...)`. I tried replacing them with a generic `getFailurePayload(StackEvent payload...)` method here, but that doesn't work.